.vercelignore

```diff
@@ -1,17 +1,19 @@
 *
 
 # general
-!utils/
+!utils
 !utils/run.js
+!utils/vercel-build
+!utils/vercel-build/**
 !.yarnrc
 !yarn.lock
 !package.json
 
 # api
-!api/
+!api
 !api/**
 
 # packages
-!packages/
```

Comment:
I think the slash is necessary since there was a but in Vercel CLI negated patterns

Reply:
CLI deploy seems fine. Might have been a bug that was since resolved (but removing the slash was necessary in this case for `vc build` to build the proper files selection 🤷).